### PR TITLE
US10_Added Delete file, updated Driver, browserUtils, FilesPage and step_definitions

### PR DESCRIPTION
**Pull request by SForogh**

### CODE REVIEW

**Review by latifaforozan — APPROVED** [review]
Looks perfect Sohaila! you did great work as always and also I like this nice collaboration how we use the same classes, locators and steps.

File: src/test/java/net/meetsky/utilities/Driver.java [hunk 1]
```diff
@@ -31,7 +32,11 @@ public static WebDriver getDriver() {
 
             switch (browserType.toLowerCase()) {
                 case "chrome":
-                    driverPool.set(new ChromeDriver());
+                    ChromeOptions options = new ChromeOptions();
+                    options.addArguments("--remote-allow-origins=*");
```
Comment (hamidalakbarov) [inline src/test/java/net/meetsky/utilities/Driver.java:36]: Thanks for update!

File: src/test/java/net/meetsky/utilities/Driver.java [hunk 2]
```diff
@@ -31,7 +32,11 @@ public static WebDriver getDriver() {
...
+                    driverPool.set(new ChromeDriver(options));
+                    driverPool.get().manage().window().maximize();
+                    driverPool.get().manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
```
Comment (hamidalakbarov) [inline src/test/java/net/meetsky/utilities/Driver.java:39]: Sohaila what is the reason you added maximize and implicit wait in here?

Comment (SForogh) [reply src/test/java/net/meetsky/utilities/Driver.java:39]: Oops, forgot that we have them separately. :) 
Please approve the merge request, Latifa is going to update it in her PR that is on the way. Thank you.

Comment (hamidalakbarov) [reply src/test/java/net/meetsky/utilities/Driver.java:39]: Got you, thank you!

**Comment by hamidalakbarov** [comment]
Great job Sohaila, thanks!

**Pull request merged**